token/program-2022/src/extension/mod.rs

```diff
@@ -397,11 +450,46 @@ pub trait BaseStateWithExtensions<S: BaseState> {
         get_extension_bytes::<S, V>(self.get_tlv_data())
     }
 
+    /// Fetch the bytes for a TLV entry, where repetitions are allowed
+    fn get_repeating_extension_bytes<V: Extension>(
+        &self,
+        repetition: usize,
+    ) -> Result<&[u8], ProgramError> {
+        get_all_extension_bytes::<S, V>(self.get_tlv_data()).map(|x| {
+            *x.get(repetition.saturating_sub(1))
+                .ok_or::<ProgramError>(ProgramError::InvalidAccountData)
+                .unwrap()
+        })
+    }
+
+    /// Fetch all bytes for each entry of a particular TLV entry, where
+    /// repetitions are allowed
+    fn get_all_extension_bytes<V: Extension>(&self) -> Result<Vec<&[u8]>, ProgramError> {
+        get_all_extension_bytes::<S, V>(self.get_tlv_data())
+    }
```

Comment:
Are these needed? `get_extension_bytes` is only used for emitting a slice of bytes for token metadata, and we won't need to do that with the group interface, so I'd prefer to remove these functions if they're not needed